README.md

```diff
@@ -28,6 +28,15 @@ cd ios
 pod install
 ```
 
+Make sure to have Internet access enabled on your Android emulator. If you can't access the browser you may need to launch your emulator with the below commands:
+
+```
+emulator -list-avds
+
+emulator @{YourEmulator} -dns-server 8.8.8.8
```

Comment:
I'm curious, are these the emulator commands you ended up using? Did you launch an emulator from CLI?

Or do you use the AVD through Android Studio?

if you could describe your workflow on this

Reply:
Done